bookwyrm/templates/lists/list.html

```diff
@@ -13,10 +13,11 @@
 
 <div class="columns mt-3">
     <section class="column is-three-quarters">
-        {% if not items.exists %}
+        {% include "snippets/booklist-pagination.html" with page=items %}
```

Comment:
Did this. It causes the pagination elements to be beneath the list items, is that desired? It would slow down pagination if you need to scroll past all of the items each time.

Reply:
Yes, that's intended. I guess my mental model of pagination is that you'd want to look through the items before paginating. Whether or not that's better is totally debatable, of course, but within the scope of this PR, I think it's better to keep things consistent across the application.